From 2afc40a1767c8dd84915d701ea0946880e5b30d5 Mon Sep 17 00:00:00 2001
From: river <riverfor@gmail.com>
Date: Sat, 11 Aug 2018 03:04:05 -0400
Subject: [PATCH] fix-default-decoding

---
 Python/bltinmodule.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Python/bltinmodule.c b/Python/bltinmodule.c
index 9ce3b27..05f50e0 100644
--- a/Python/bltinmodule.c
+++ b/Python/bltinmodule.c
@@ -22,7 +22,7 @@ const char *Py_FileSystemDefaultEncoding = "mbcs";
 #elif defined(__APPLE__)
 const char *Py_FileSystemDefaultEncoding = "utf-8";
 #else
-const char *Py_FileSystemDefaultEncoding = NULL; /* use default */
+const char *Py_FileSystemDefaultEncoding = "utf-8"; /* use default */
 #endif
 
 /* Forward */
-- 
2.7.4

