From 6c660a9f8110a281da13b1f20f756fe667ee88af Mon Sep 17 00:00:00 2001
From: River <riverfor@gmail.com>
Date: Mon, 7 May 2018 16:19:22 +0800
Subject: [PATCH] update configure.ac

---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fc1dba6b02..6f41d8a30e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -919,12 +919,13 @@ if $CPP $CPPFLAGS conftest.c >conftest.out 2>/dev/null; then
   _arm_arch=`sed -n -e '/__ARM_ARCH/d' -e 's/^arm_arch = //p' conftest.out`
   AC_MSG_RESULT([$ANDROID_API_LEVEL])
   AC_DEFINE_UNQUOTED(ANDROID_API_LEVEL, $ANDROID_API_LEVEL, [The Android API level.])
+  AC_DEFINE_UNQUOTED(__ANDROID__, 1, [The Android flag.])
 
   AC_MSG_CHECKING([for the Android arm ABI])
   AC_MSG_RESULT([$_arm_arch])
   if test "$_arm_arch" = 7; then
     BASECFLAGS="${BASECFLAGS} -mfloat-abi=softfp -mfpu=vfpv3-d16"
-    LDFLAGS="${LDFLAGS} -march=armv7-a -Wl,--fix-cortex-a8"
+    LDFLAGS="${LDFLAGS} -march=armv7-a "
   fi
 else
   AC_MSG_RESULT([not Android])
-- 
2.11.0 (Apple Git-81)

